Contents/mods/Hydrocraft/media/scripts/Herbalism.txt

```diff
@@ -168,6 +181,7 @@ item HCNettleteamug
         Lipids 			= 	0,
         Calories 		= 	10,
 	CantBeFrozen		=	TRUE,
+	DisplayCategory          =   FoodB,
```

Comment:
I think this is primarily for food sickness treatment. So medical display category. same for other nettles stuff